### ISSUE TITLE
Argument and its type joined in docs

### ISSUE THREAD

**Issue by OriolAbril** [pydata/xarray#3056]
This issue has nearly no effect on users, but (at least for me) it is quite bothering aesthetically. It also looks like something simple to solve even though I am not an sphinx expert so I am not sure. 

In the docstring, there is a colon between the name of one argument and its type, however, in the online documentation, the name and the type are smashed together. In the case of `xr.align`:

![image](https://user-images.githubusercontent.com/23738400/60357722-b3ca3200-99d4-11e9-8407-b4a823626b7b.png)
![image](https://user-images.githubusercontent.com/23738400/60357752-cba1b600-99d4-11e9-8abe-3f4e6e32db44.png)
![image](https://user-images.githubusercontent.com/23738400/60357768-d4928780-99d4-11e9-8754-e2d9e9fe6039.png)

I think it would be more readable and appealing if there was some kind of separation between the two.

**Comment by shoyer** [comment]
I agree, this is definitely not ideal.

It's definitely an upstream issue with the theme we use for our docs -- possibly https://github.com/rtfd/sphinx_rtd_theme/issues/750?

It would be great if you (or anyone else) wants to look into this.

**Comment by OriolAbril** [comment]
It is an issue of incompatibilities with sphinx_rtd_theme and sphinx2, and I am not sure when will it be solved, I have looked a little into it but for now I don't know how. There is one small problem with xarray doc generation behaviour. In the `doc/environment.yml` the sphinx version is fixed to 1.8, however, in the rtd build, the `doc/enviroment.yml` is overridden and sphinx 2 is used. I generated the docs locally with the environment in `doc/` (right) to compared with the online docs (left):

![image](https://user-images.githubusercontent.com/23738400/60362089-4ae8b700-99e0-11e9-9ef6-6e998779fb2b.png)

I have no clear preference between sphinx1 or sphinx2, but both online and local docs should be the same. If you prefer sphinx2, I have no problem modifying `doc/environment.yml`, otherwise, I don't think a pull request can solve it (the build script in in the [rtd website](https://readthedocs.org/projects/xray/builds/9296291/) does not look like the one on github).
